samples/storageclass/powermax_zone.yaml

```diff
@@ -0,0 +1,18 @@
+---
+# This minimal YAML represents a StorageClass which can be used with
+# PowerMax zones. Any missing parameters such as SYMID can be provided
+# in the secret for the array. Consult the CSM Documentation for more
+# details.
+
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: powermax-zone
+parameters:
+  csi.storage.k8s.io/fstype: xfs
```

Comment:
Do we need to comment optional parameters here as an example?

Reply:
I wanted to present a clean concise sample. The other samples already indicate optional parameters if they are contained in the secret.